### PR TITLE
added units test for functions in mapper/thrift/replicator.go

### PR DESCRIPTION
**Pull request by d-vignesh**
**What changed?**
Added unit test for function in common/types/mapper/thrift/replicator.go

**Why?**
Improve code coverage

**How did you test it?**
local testing with go test.

### CODE REVIEW

File: common/types/testdata/replication.go
Comment (Groxx) [inline common/types/testdata/replication.go:147]: hmm.  I guess this file existing is a decent argument that we do need a random-data factory :\

oh well.  additions look reasonable 👍

File: common/types/mapper/thrift/replicator_test.go
Comment (Groxx) [inline common/types/mapper/thrift/replicator_test.go:236]: a thing we've been kinda trying to shift towards / for minor test-boilerplate improvement, making the test container a `map[string]` can be convenient:
```suggestion
	testCases := map[string]*types.GetDomainReplicationMessagesRequest{
		"non-nil": &testdata.GetDomainReplicationMessagesRequest,
		"empty":   &types.GetDomainReplicationMessagesRequest{},
		"nil":     nil,
	}
	for name, tc := range testCases {
		t.Run(name, func(t *testing.T) {
```
we obviously mostly don't do this though, your `[]struct` is overwhelmingly the pattern in these tests, so this is not at all a request to change anything.  just an observation for anyone reading / reviewing / etc :)

File: common/types/testdata/replication.go
Comment (Groxx) [inline common/types/testdata/replication.go:30]:
```suggestion
	MaximumPageSize = int32(5)
```

**Review by Groxx — APPROVED** [review]
some very minor "might be nicer if..." thoughts while reading, but overall 👍 nice and simple, big jump in coverage, thanks!

if you _want_ to make any changes I can hold off on merging, but this looks more than good enough to me as-is.
or I'll just merge for you tomorrow if I don't hear back :)

File: common/types/mapper/thrift/replicator_test.go
Comment (d-vignesh) [reply common/types/mapper/thrift/replicator_test.go:236]: sure @Groxx will follow the map pattern for future tests.

**Comment by d-vignesh** [comment]
Hi @Groxx, thank you for the review, if things look fine from your end, we can go ahead and merge it.

**Comment by Groxx** [comment]
> Hi @Groxx, thank you for the review, if things look fine from your end, we can go ahead and merge it.

ah, maybe you don't have a merge button after approval.
oh well.  I'll do that now :)

**Pull request merged**
